### PR TITLE
add list of brands using RN

### PR DESCRIPTION
**Pull request by codyzu**
Added a list of brands in the community and clients using RN

### CODE REVIEW

**Comment by codyzu** [comment]
Not sure where to put these slides or if we should add logos. TBD.

File: slides/slides.md [hunk 1]
```diff
@@ -80,3 +80,33 @@ class: dense
...
+* Many more...
+
+---
+class: dense
+---
+
+# Who's using it? - NearForm Partners
+
+* HSE
+* Aer Lingus
```
Comment (codyzu) [inline slides/slides.md:107]:
```suggestion
* Aer Lingus
```

File: slides/slides.md [hunk 2]
```diff
@@ -80,3 +80,33 @@ class: dense
...
+* Virgin Media
+* Telus
+* Get Ireland
```
Comment (codyzu) [inline slides/slides.md:110]:
```suggestion
* Get Ireland Walking
```

File: slides/slides.md [hunk 3]
```diff
@@ -80,3 +80,32 @@ class: dense
...
 https://reactnative.dev/docs/environment-setup
+
+---
+class: dense
+---
+
+# Who's using it? - Community
+
+* Meta: Facebook, Oculus, Messenger Desktop
+* Microsoft: Office, Outlook, Teams, Xbox Game Pass, Skyp
```
Comment (SociableSteve) [inline slides/slides.md:91]: Typo in Skype

File: slides/slides.md [hunk 4]
```diff
@@ -80,3 +80,32 @@ class: dense
...
+* Shopify
+* Wix
+* Pinterest
+* Tesla
+* Discord
+* Puma
+* Many more...
```
Comment (SociableSteve) [inline slides/slides.md:98]: Is there anything that this `many more...` can link to?

File: slides/slides.md [hunk 5]
```diff
@@ -80,3 +80,32 @@ class: dense
...
+
+---
+class: dense
+---
+
+# Who's using it? - NearForm Partners
```
Comment (SociableSteve) [inline slides/slides.md:104]: Not sure this is the best title - this reads as people we've worked with who are using it, not necessarily projects we have worked on.

**Review by SociableSteve — CHANGES_REQUESTED** [review]

File: slides/slides.md [hunk 6]
```diff
@@ -80,3 +80,31 @@ class: dense
...
+|Discord| <img src="/images/discord-logo.svg" width=40> |
+| Many more... ||
+
+---
+class: dense
+---
+
+# Who's using it? - NearForm Partners
+
+* Aer Lingus
```
Comment (AlanSl) [inline slides/slides.md:105]:
```suggestion
```

**Review by AlanSl — CHANGES_REQUESTED** [review]
See comment on Slack!

**Comment by SociableSteve** [comment]
Current slides:

image
image

**Review by robwalkerco — APPROVED** [review]
Looks good to me 🚢

**Pull request merged**